src/main/java/bit/javaoop/managersPackage/Manager.java

```diff
@@ -0,0 +1,33 @@
+package bit.javaoop.managersPackage;
+
+import bit.javaoop.employeesPackage.AbstractEmployee;
+import bit.javaoop.employeesPackage.Employee;
+
+import java.util.ArrayList;
+
+
+public abstract class Manager extends AbstractEmployee {
+    protected double budget;
+    protected double employeesLimit;
+    private double budgetLeft = budget;
+
+    public Manager(String name, double salary, double budget, double employeesLimit) {
+        super(name, salary);
+        this.budget = budget;
+        this.employeesLimit = employeesLimit;
+    }
+
+    public static ArrayList<Employee> employeesList;
+
+
+    public abstract boolean canHireEmployee();
+
+    public void hireEmployee(Employee employee) {
+        employeesList.add(employee);
+        budgetLeft -= employee.getSalary();
+        employeesLimit -= 1;
+        Employee employee2 = new Employee(employee.getName(), employee.getSalary());
+    }
+
+
+}
```

Comment:
![image](https://user-images.githubusercontent.com/18233796/38771051-65031334-401c-11e8-9942-39f5847bf8f7.png)
Po poprawce a implementacje tej klasy czyli BudgetManager i EmployeeManager powinny zaimplementować już metodę hire na swój sposób:) i tak samo jak SATISFYING_SALARY powinny to być stałe